metaci/build/tests/test_flows.py

```diff
@@ -198,6 +212,79 @@ def test_post_task__multiple_robot_tasks(get_spec):
         assert 3 == TestMethod.objects.all().count()
 
 
+@responses.activate
+@pytest.mark.django_db
+def test_post_task_gus_bus_test_results_enabled(get_spec, mocker, mocked_responses):
+    """Test for scenario where there are multiple Robot tasks defined
+    in a single flow. We want to make sure that test results and related
+    assets are created and associated with the correct related objects and that
+    the proper api endpoint is called."""
+    mocker.patch(
+        "metaci.build.flows.settings",
+        METACI_RELEASE_WEBHOOK_URL="https://webhook",
+        METACI_RELEASE_WEBHOOK_ISSUER="MetaCI",
+        METACI_RELEASE_WEBHOOK_AUTH_KEY="test",
+        DJANGO_TIME_ZONE="US/Pacific",
+        RESULT_EXPORT_ENABLED=True,
+    )
+    mocker.patch(
+        "metaci.build.tests.test_flows.settings",
+        METACI_RELEASE_WEBHOOK_URL="https://webhook",
+    )
+    mocker.patch(
+        "metaci.testresults.robot_importer.settings",
+        METACI_RELEASE_WEBHOOK_URL="https://webhook",
+        METACI_RELEASE_WEBHOOK_ISSUER="MetaCI",
+        METACI_RELEASE_WEBHOOK_AUTH_KEY="test",
+    )
+    mocker.patch(
+        "metaci.release.utils.settings",
+        METACI_RELEASE_WEBHOOK_ISSUER="https://webhook",
+        METACI_RELEASE_WEBHOOK_AUTH_KEY="12345",
+    )
```

Comment:
Does it work to patch django.conf.settings instead of patching settings objects in a bunch of different modules? I'm not sure if it will, but if there is a singleton settings object which gets initialized once, it might

Reply:
Also, if there are settings that should be set to a single value for all tests that is different from the non-test default, I would do that in https://github.com/SFDO-Tooling/MetaCI/blob/main/config/settings/test.py rather than using mocks.